qemu/tests/cfg/nvdimm.cfg

```diff
@@ -41,6 +41,8 @@
 
     variants:
         - nvdimm_basic:
+            ppc64le, ppc64:
+                depends_pkgs = "ndctl"
```

Comment:
`ndctl` should be the basic package for nvdimm test, not only for PPC.

Reply:
Updated.

Reply:
(1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.ppc64le.io-github-autotest-qemu.nvdimm.nvdimm_basic: STARTED
 (1/1) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.ppc64le.io-github-autotest-qemu.nvdimm.nvdimm_basic: PASS (250.56 s)

Can you help to review it again ? Thanks a lot @PaulYuuu